pkg/appfile/helm/schema_test.go

```diff
@@ -0,0 +1,57 @@
+package helm
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestGenerateSchemaFromValues(t *testing.T) {
```

Comment:
good job, it'll be better if we can have more cases with small different schema cases file.

Reply:
Yes, that's what we really need. The current `yaml=>cue=>openapi` approach may fail for special cases we haven't tested.